pyproject.toml

```diff
@@ -107,11 +108,20 @@ warn_unused_ignores = true
 warn_return_any = true
 warn_unused_configs = true
 
+[[tool.mypy.overrides]]
```

Comment:
No need to fight `mypy` while still in heavy development mode. We can enable this later as it is not user facing anyway.